contrib/server/operations/pbench-upload-results.py

```diff
@@ -52,32 +51,27 @@ def get_md5(tarball: Path) -> str:
 
 def upload(
     server: str, token: str, tarball: Path, metadata: Optional[list[str]] = None
-) -> Response:
-    query_parameters = {}
-
+) -> requests.Response:
     md5 = get_md5(tarball)
-    query_parameters["access"] = "public"
-    dataset = tarball.name
-    satellite, _ = tarball.parent.name.split("::")
-    meta = [
-        f"global.server.legacy.migrated:{datetime.datetime.now(tz=datetime.timezone.utc):%Y-%m-%dT%H:%M}"
-    ]
-    if satellite:
+    uploaded = datetime.datetime.fromtimestamp(
+        tarball.stat().st_mtime, tz=datetime.timezone.utc
+    )
+    meta = [f"global.server.legacy.migrated:{uploaded:%Y-%m-%dT%H:%M}"]
+    if "::" in tarball.parent.name:
+        satellite, _ = tarball.parent.name.split("::")
```

Comment:
Just for full paranoia's sake, I think you should provide the second argument to `split()` so that we only get two values back.

Reply:
The only way we'll ever have `::` is if it's a satellite transfer, and "there can be only one". Being paranoid for a special purpose ops utility just didn't seem worthwhile.